openke/config/Tester.py

```diff
@@ -199,6 +205,24 @@ def run_ans_prediction(self, topk, outfile_name, dyntop, mode):
                 record['tail_predictions_fil']['correctness'] = truths_tail_fil.astype(int).tolist()
 
             test_data.append(record)
+        runtime = time.time() - start_time
+
+        hitsHead = 0
+        hitsTail = 0
+        for data in test_data:
+            print(data['head_predictions_raw']['correctness'])
+            print(data['tail_predictions_raw']['correctness'])
+            if 1 in data['head_predictions_raw']['correctness']:
```

Comment:
use `head_predictions_fil` and `tail_predictions_fil` to compute the recall because it uses the test data only